### ISSUE TITLE
TypeError: undefined. attachedTemplates is not iterable

### ISSUE THREAD

**Issue by Aedif** [caewok/fvtt-walled-templates#77]
Prevents pretty much all interaction with a token.
Appears to be a conflict between Walled Templates and Token Attacher that occurs on tokens that had their Actors deleted.

Steps to reproduce (tried this in fresh world too):

1. Activate Token Attacher and Walled Templates
2. Create an Actor and drag out a token onto the scene
3. Delete the placed token's Actor
4. Attempt to drag the token

![image](https://github.com/caewok/fvtt-walled-templates/assets/7693704/4292a17d-ba2b-40fd-96f7-0b80ecc911c9)

**Comment by Aedif** [comment]
I tested this again and this does not appear to be a conflcit between TA and Walled Templates. Walled Template simply doesn't handle Tokens without Actors properly.

Line marked as `Issue #65` should be changed to return an empty array.

```js
function attachedTemplates() {
  if (!this?.actor?.effects) return; // Issue #65.
  return this.actor.effects
    .filter((e) => e.origin && e.origin.includes('MeasuredTemplate'))
    .map((e) => fromUuidSync(e.origin)?.object)
    .filter((t) => Boolean(t)); // Drop undefined templates
}
```
>>>

```js
function attachedTemplates() {
  if (!this?.actor?.effects) return []; // Issue #65.
  return this.actor.effects
    .filter((e) => e.origin && e.origin.includes('MeasuredTemplate'))
    .map((e) => fromUuidSync(e.origin)?.object)
    .filter((t) => Boolean(t)); // Drop undefined templates
}
```

**Comment by tirzah2** [comment]
Thumbs up

**Comment by caewok** [comment]
Should be fixed in v0.7.0. Thanks!
